udp-ballerina/build.gradle

```diff
@@ -113,9 +113,9 @@ task copyStdlibs(type: Copy) {
 
 task updateTomlFile {
     doLast {
-        def stdlibDependentIoVersion = project.stdlibIoVersion.split("-")[0]
-        def stdlibDependentLogVersion = project.stdlibLogVersion.split("-")[0]
-        def nettyVersion = project.nettyVersion.split("-")[0]
+        def stdlibDependentIoVersion = project.stdlibIoVersion.replace("${snapshotVersion}", "")
+        def stdlibDependentLogVersion = project.stdlibLogVersion.replace("${snapshotVersion}", "")
+        def nettyVersion = project.nettyVersion.replace("${snapshotVersion}", "")
```

Comment:
We don't need to replace `SNAPSHOT` in netty version. It should be a stable version always.

Reply:
removed `replace(..`